HIP/hip-410.md

```diff
@@ -168,49 +215,49 @@ dropped when converted to tinyBars.
-devnet, and 0x12a for local networks). Future evolution in the Ledger ID may
-impact this.
+For all transaction types the `chainID` specified must match the `chainID` of the
+network as specified in [HIP-26](https://hips.hedera.com/hip/hip-26)
+(`0x127` for mainnet, `0x128` for testnet, `0x129` for devnet, and `0x12a` for
+local networks).
 
 ### Transaction Signing Requirements
 
-The sender of an ethereum transaction body will be the account with an account
+The sender of an Ethereum transaction body will be the account with an account
 alias that is an ECDSA(secp256K1) public key. To identify the account the public
-key will be recovered from the ethereum transaction, and converted to an
-ethereum address. The transaction sender will then be set to the account whose
-ethereum address of the alias key has the same ethereum address.
+key will be recovered from the Ethereum transaction, and converted to an
+Ethereum address. The transaction sender will then be set to the account whose
+Ethereum address of the alias key has the same Ethereum address.
 
-For the purposes of authentication signing the ethereum transaction will provide
+For the purposes of authentication signing the Ethereum transaction will provide
 the same authorization as though the key signed the entire Hedera transaction.
 
 ### Paying for the Transaction
 
 In addition to the account signing the Ethereum transaction (the sender account)
 each Hedera transaction containing the Ethereum transaction will also need to be
-signed by a hedera account (the relayer account). These may be the same account
+signed by a Hedera account (the relayer account). These may be the same account
 or different accounts.
 
 If the relayer and sender account are the same account there are no limitations
 on hbar transfers.
 
-When the relayer and sender account is different the following restrictions
+When the relayer and sender account are different the following restrictions
 apply to hbar transfers.
 
-- The effective gas cost is either the gasUsed field (if a contract call or
+- The effective gas cost is either the `gasUsed` field (if a contract call or
   contract create transaction) or the gas that a system contract would charge
   for the operation
 - The gas price is either the `gasPrice` field for legacy and type 1 (EIP-2930)
   transactions or the `max_fee_per_gas` field for type 2 (EIP1339 transactions).
   This value is expressed in weibars.
 - The sender will be debited hbar equal or less than the gas price multiplied by
   the effective gas cost (the sender portion).
-- If the sender portion is greater than the total fees the relayer will receive
+- If the sender portion is greater than the total fees the relayer will receive,
   the excess or the sender portion should be reduced. If the sender portion is
   less than the total fees the sender will pay the difference.
 - If there is a `max_priority_fee_per_gas` specified then the relayers account
```

Comment:
Can you confirm that my assumption is true:

0. Fee schedule is configured to be 47 tinybars per gas, which is 470 Gwei per gas (this can be considered the base fee)
1. `User X` submits `ContractCall` TX with `maxFeePerGas=1000 Gwei`
2. Hedera nodes execute the TX and charge `470Gwei per gas`. The rest (530 gwei per gas) is sent to the relayer